formats/shared/definitions/_specialist_document.jsonnet

```
@@ -1955,6 +1958,286 @@
       },
     },
   },
+  research_for_development_output_metadata: {
+    type: "object",
+    additionalProperties: false,
+    properties: {
+      bulk_published: {
+        type: "boolean",
+      },
+      fcdo_review_status: {
```

```suggestion
      review_status: {
```

I would if we could make all these fields that reference `fcdo` generic so we don't have to deal with this in the future... I know they're not presented on the frontend, but if we're going to change them from `dfid_` I feel like we may as well make it generic now.